InventorySales/etc/di.xml

```diff
@@ -130,10 +132,15 @@
             </argument>
         </arguments>
     </virtualType>
-    <type name="Magento\InventorySales\Model\CheckItemsQuantity">
+    <virtualType name="AreProductsSalableForRequestedQtyConditionChainOnPlaceOrder" type="Magento\InventorySales\Model\AreProductsSalableForRequestedQty">
```

Comment:
Is it possible to avoid virtual type here?

Reply:
@sidolov, removed